test/util/testapp_upgrade.go

```diff
@@ -0,0 +1,56 @@
+package util
```

Comment:
[not blocking] FYI: https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common